README.md

```diff
@@ -803,6 +813,65 @@ $ ulimit -u # processes / threads
 
 Just pass a new number as the argument to change it.
 
+## Prometheus Support
+
+Vegeta has a built-in Prometheus Exporter that may be enabled during "attacks" so that you can point any Prometheus instance to Vegeta instances and get some metrics about http requests performance and about Vegeta process itself.
```

Comment:
```suggestion
Vegeta has a built-in Prometheus Exporter that may be enabled during "attacks" so that you can point any Prometheus instance to Vegeta instances and get some metrics about http requests performance and about the Vegeta process itself.
```